landing-page/index.html

```diff
@@ -6,7 +6,6 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>React-Admin - The Open-Source Framework For B2B Apps</title>
 
-    <link rel="modulepreload" crossorigin href="/src/main.ts" />
```

Comment:
This line was causing the issue. The `/src/main.ts` script was already included later in the page.